src/app/components/navigation/navigation.component.scss

```diff
@@ -14,6 +14,77 @@
   font-weight: 500;
 }
 
+.navbar-desktop .navigation-section-link {
+  position: relative;
+
+  &:before {
```

Comment:
I do notice that there are some CSS properties that are repeated. We can use `@mixin`, which is a sass utility to define the styles that can be reused in the CSS file. Here is an example: 

```
@mixin exampleName($backgroundColor, $top){
    content: "";
    width: 0;
    height: 5px;
    background-color: $backgroundColor;
    position: absolute;
    top: $top;
    left: 0;
    transition: all 0.5s; 
}
```
and then use `@include` to call the @mixin function and pass the arguments

Here is the reference: https://sass-lang.com/documentation/at-rules/mixin

Also, I implemented @mixin  and @include in this file (check lines 10-15, and then lines 99 and 132).

Reply:
Great, I will look into this and make the appropriate changes (have been meaning to learn sass next anyway)

and no problem, please take your time

Reply:
Alright, I made the required changes, I think it roughly saved about 20 lines of code, do I open up a new pull request? I am not sure how this works as well

Reply:
No need to create a new PR. The changes you have made are already saved in this PR. This looks good to me!